src/emulator/dataconnectEmulator.ts

```diff
@@ -170,6 +179,22 @@ export class DataConnectEmulator implements EmulatorInstance {
     return Emulators.DATACONNECT;
   }
 
+  async clearData(): Promise<void> {
```

Comment:
Can one still clear the DB even if running a separate postgres? I'd say yes, but any concerns?

Reply:
That's a good product q - I think it's reasonable to offer, but if they're connecting to a real DB that becomes a pretty scary big red button.

Reply:
But given the code as it is written right now, this seems to noop if it's not using pglite.

I think it should either do what it says on the label, or fail with a clear error message (e.g. "unimplemented"), but not noop.